CodeStarterCamp_Week4/FitnessCenter.swift

```diff
@@ -0,0 +1,133 @@
+//
+//  FitnessCenter.swift
+//  CodeStarterCamp_Week4
+//
+//  Created by 정선아 on 2022/05/10.
+//
+
+import Foundation
+
+enum InputError: Error {
+    case empty
+    case outOfValue
+    case invaildValue
+}
+
+class FitnessCenter {
+    let name: String
+    var targetBodyCondition: BodyCondition
+    var member: Person?
+    var listOfRoutine: [Routine]
+    
+    init(name: String, targetBodyCondition: BodyCondition, member: Person, listOfRoutine: [Routine]) {
+        self.name = name
+        self.targetBodyCondition = BodyCondition()
+        self.member = member
+        self.listOfRoutine = listOfRoutine
+        
+    }
```

Comment:
프로퍼티 4개 중에서, 2개는 기본값을 설정해놨기 때문에,
`이니셜라이저`에서는 2개의 프로퍼티(name, listOfRoutine)만 할당해주면 됩니다. 😄 

```suggestion
    init(name: String, listOfRoutine: [Routine]) {
        self.name = name
        self.listOfRoutine = listOfRoutine
    }
```

근데 왜 하필 member 랑 bodyGoal 만 기본값을 줬을까요?
그 이유는.. 이 2개 프로퍼티는 파라미터로 값을 전달 받는 게 아니라, 타입 내부에서 `readLine()` 메서드를 통해, 유저한테 값을 입력받아야 하기 때문이에요..!

추가 코멘트를 읽다 보면, 이해가 좀 되실 겁니다!

Reply:
이렇게 되면 제가 하던 고민들도 싹 해결이네요!
피트니스 센터 이름과 루틴은 인스턴스 생성할 때 파라미터에 값이 넣을 수 있도록, 해당 프로퍼티들만 이니셜라이저하고
readLine()에서 입력받을 프로퍼티들은 그럴 필요가 없게 되네요...!!

저도 이렇게 유연한 사고를 가지고 싶어요...😞
막상 프로그래밍할 때 너무 돌아가는 방법만 채택해서 사용하는 거 같아요🙄

Reply:
저도 솜의 코드를 읽으면서 1시간 이상 고민해서 찾은 방법입니다.. ㅋㅋㅋ
절대 훅훅 떠오르지 않아요..!! 자신감 가지세요 😄